src/core/errorHandler.js

```diff
@@ -140,6 +167,29 @@ export function withErrorHandler({ name, id }) {
 }
 
 /*
+ * This is a page level error decorator. It works like the `withErrorHandler()`
+ * decorator, but aims at synchronizing both the server and client sides and
+ * should be used for page level components.
+ *
+ * Pass the optional `extractId` function to create "per unique page" level
+ * error handlers. This function takes the component's props and must returns a
+ * unique value based on these props (e.g., based on the `slug`, `uniqueId`,
+ * etc.).
+ *
+ * When this optional function is not supplied, we "fix" the error handler `id`
+ * (e.g., `SomeComponentPage`) otherwise we pass the `name` suffixed by `Page`,
+ * and the HOC will create the final error handler ID thanks to `extractId`.
+ */
+export const withPageErrorHandler = ({ name, extractId = null }) => {
```

Comment:
I think the `extractId` approach will solve a lot of client/server problems and not just for pages. For example, an error handler for each review in the list of reviews could have an `extractId()` that returns the ID of the review object.

Thus, we should call this something different.

* `withFixedErrorHandler()` ?
* `withMemoizedErrorHandler()` ?
* `withKeyedErrorHandler()` ?

Reply:
`withFixedErrorHandler()`